app/views/audit.scala.html

```diff
@@ -995,10 +995,7 @@ <h1 class="text-center bold">Surface Problem</h1>
                 url: surveyUrl,
                 type: 'post',
                 dataType: 'json',
-                data: JSON.stringify({
-                    "answered_questions": formData,
-                    "is_admin": equals(@user.get.role.getOrElse(""), "Administrator")
```

Comment:
Hey @maddalihanumateja, do you remember why you needed the admin check here? I want to make sure by removing it, we aren't breaking anything.